src/rgw/rgw_service.cc

```diff
@@ -53,6 +53,12 @@ int RGWServices_Def::init(CephContext *cct,
   bucket_sobj = std::make_unique<RGWSI_Bucket_SObj>(cct);
   bucket_sync_sobj = std::make_unique<RGWSI_Bucket_Sync_SObj>(cct);
   bi_rados = std::make_unique<RGWSI_BucketIndex_RADOS>(cct);
+  // Q: will we have the bucket layout available here?
+  // A: metadata log uses the `neorados` along the path to determine
```

Comment:
We won't be using neorados in this run, though. The current plan is to use the interface in cls_fifo_legacy for ease of backporting, and then it can be updated to neorados for better Aio support later.